packages/manager/modules/support/src/tickets/new-ticket/new-ticket.constant.js

```diff
@@ -0,0 +1,17 @@
+export const ISSUE_TYPES = {
+  ID_1: 34602,
+  ID_2: 1316,
```

Comment:
**nitpick:** Can you use meaningful variable name? `ID_1` and `ID_2` doesn't signify anything. Maybe the type of issues can make a good indicator

Reply:
Done Anoop.